asset_stream_manager/main.cc

```diff
@@ -113,17 +117,23 @@ ABSL_FLAG(int, manifest_updater_threads, 4,
           "Number of threads used to compute file hashes on the workstation.");
 ABSL_FLAG(int, file_change_wait_duration_ms, 500,
           "Time in milliseconds to wait until pushing a file change to the "
-          "instance after detecting it.");
+          "instance after detecting it");
 ABSL_FLAG(bool, check, false, "Check FUSE consistency and log check results");
 ABSL_FLAG(bool, log_to_stdout, false, "Log to stdout instead of to a file");
 ABSL_FLAG(cdc_ft::JedecSize, cache_capacity,
           cdc_ft::JedecSize(cdc_ft::DiskDataStore::kDefaultCapacity),
-          "Cache capacity. Supports common unit suffixes K, M, G.");
+          "Cache capacity. Supports common unit suffixes K, M, G");
 ABSL_FLAG(uint32_t, cleanup_timeout, cdc_ft::DataProvider::kCleanupTimeoutSec,
           "Period in seconds at which instance cache cleanups are run");
 ABSL_FLAG(uint32_t, access_idle_timeout, cdc_ft::DataProvider::kAccessIdleSec,
           "Do not run instance cache cleanups for this many seconds after the "
           "last file access");
+ABSL_FLAG(std::string, config_dir, "",
+          "Directory on the host to stream from with the json configuration "
+          "for asset stream manager stored in the file "
+          "`assets_stream_manager.json`");
+ABSL_FLAG(std::string, log_dir, "",
```

Comment:
We should define a reasonable default log dir, e.g. %APPDATA%\cdc-file-transfer\logs.

I just realized we also need to change the local cache path in MultiSession::GetCachePath(), which is hardcoded to %APPDATA%\GGP\asset_streaming. This needs to be configurable as well. But this can be done in a separate CL.

Reply:
Ok, I will change that as well separately.